### ISSUE TITLE
使用imageListToImageBatch节点图片多的时候效率较低

### ISSUE THREAD

**Issue by civen-cn** [yolain/ComfyUI-Easy-Use#492]
4090跑的，
直接torch.cat(images, dim=0)的用时仅3s
![image](https://github.com/user-attachments/assets/49c398b7-015e-41f4-a02d-eb98f05e3e54)

逐个cat用了好久都出不来。貌似内存没有增（上次我出的图可能还真存在大小不一的情况），就是超级慢
![image](https://github.com/user-attachments/assets/dd7598b9-a133-4790-8bb9-79588066043c)

增加计数打印情况：每100 +17s
[2024-10-25 16:24:19.736] 0
9s
[2024-10-25 16:24:28.295] 100
25s
[2024-10-25 16:24:53.187] 200
41s
[2024-10-25 16:25:34.383] 300
57s
[2024-10-25 16:26:31.878] 400
73s
[2024-10-25 16:27:45.658] 500
90s
[2024-10-25 16:29:15.822] 600
107s
[2024-10-25 16:31:02.193] 700
122s
[2024-10-25 16:33:04.802] 800
139s
[2024-10-25 16:35:23.646] 900
155s
[2024-10-25 16:37:58.762] 1000
172s
[2024-10-25 16:40:50.775] 1100
189s
[2024-10-25 16:43:59.917] 1200
207s
[2024-10-25 16:47:26.480] 1300
[2024-10-25 16:51:10.136] 1400
[2024-10-25 16:55:11.007] 1500
[2024-10-25 16:59:41.836] 1600

_Originally posted by @civen-cn in https://github.com/yolain/ComfyUI-Easy-Use/issues/421#issuecomment-2437235478_

**Comment by civen-cn** [comment]
新版用时现仅为8s-11s之间，nice！
